ice40/devices/top-routing-virt/CMakeLists.txt

```diff
@@ -29,14 +29,17 @@ set(TIMING_IMPORT_CMD "${PYTHON3} ${ICE40_IMPORT_TIMING} --read_sdf ${SDF_FILE}
 add_file_target(FILE ${SDF_FILE} GENERATED)
 append_file_dependency(DEPS ${SDF_FILE})
 
+set(UPDATE_TILES "${symbiflow-arch-defs_SOURCE_DIR}/utils/update_arch_tiles.py")
+set(UPDATE_TILES_CMD "${PYTHON3} ${UPDATE_TILES} --in_xml /dev/stdin --out_xml /dev/stdout")
```

Comment:
You probably need to delay the deref of `${PYTHON3}`, so change `${PYTHON3}` to `\${PYTHON3}`

Reply:
In this case it is not required, but you have to ensure that each place you moved a script, the `PYTHON3` was defined, or delay the deref.

Reply:
Great, thanks, this explains the issue. I'll fix that